lib/crowbar/client/command/upgrade/database.rb

```diff
@@ -50,7 +50,7 @@ def execute
                 err request.parsed_response["error"]
               end
 
-              response = JSON.parse(request.body)
+              response = JSON.parse(request.body) unless request.body.nil?
```

Comment:
@nicolasbock did you run into an error somewhere about that? would be good to see the details if you still have it.

I think the issue that you describe already got resolved by https://github.com/crowbar/crowbar-client/commit/8358869e124b02c4c1c01f814f3c528d6344da4f

because once it runs into `err` it will exit immediately. and if there is no `err` we can safely assume that there is a json body (see crowbar-init api)

other than that, maybe you missed it but the response variable would have to be set here because it is still used later on. but I think this is not relevant here, as it will exit anyways.